src/docfx/build/context/BuildScope.cs

```diff
@@ -25,17 +26,22 @@ internal class BuildScope
 
         public RedirectionMap Redirections { get; }
 
-        public BuildScope(ErrorLog errorLog, Docset docset, TemplateEngine templateEngine)
+        public Docset Docset { get; }
```

Comment:
Can this be private?

```suggestion
        private readonly Docset _docset;
```